city_scrapers/spiders/losca_metro_transit.py

```diff
@@ -0,0 +1,126 @@
+from city_scrapers_core.constants import COMMITTEE, BOARD, CITY_COUNCIL, NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import LegistarSpider
+
+
+class LoscaMetroTransitSpider(LegistarSpider):
+    name = "losca_metro_transit"
+    agency = "Los Angeles Metro Transit"
+    timezone = "America/Los_Angeles"
+    start_urls = ["https://metro.legistar.com/Calendar.aspx"]
+    # Add the titles of any links not included in the scraped results
+    link_types = []
+
+    def parse_legistar(self, events):
+        """
+        `parse_legistar` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+
+        for event in events:
+            meeting = Meeting(
+                title=event.get("Name", {}).get("label", "No Title"),
+                description=self._parse_description(event),
+                classification=self._parse_classification(event),
+                start=self.legistar_start(event),
+                end=self._parse_end(event),
+                all_day=self._parse_all_day(event),
+                time_notes=self._parse_time_notes(event),
+                location=self._parse_location(event),
+                links=self._parse_links(event),
+                source=self.legistar_source(event),
+            )
+
+            meeting["status"] = self._parse_status(meeting)
+            meeting["id"] = self._parse_id(meeting)
+
+            yield meeting
+
+    def _parse_description(self, item):
+        """Parse or generate meeting description."""
+        return ""
+
+    def _parse_classification(self, item):
+        """Parse or generate classification from allowed options."""
+        name_label = item.get("Name", {}).get("label", "").lower()
+        if "committee" in name_label:
+            return COMMITTEE
+        if "board" in name_label:
+            return BOARD
+        if "council" in name_label:
+            return CITY_COUNCIL
+        return NOT_CLASSIFIED
+
+    def _parse_end(self, item):
+        """Parse end datetime as a naive datetime object. Added by pipeline if None"""
+        return None
+
+    def _parse_time_notes(self, item):
+        """Parse any additional notes on the timing of the meeting"""
+        return ""
+
+    def _parse_all_day(self, item):
+        """Parse or generate all-day status. Defaults to False."""
+        return False
```

Comment:
It seems these functions are just wrappers for returning hardcoded values like `None`, `""`, and `False`. We can just add those same values directly to the new `Meeting` object on L23 and get rid of these wrapper functions that don't add any logic.